examples/miscellaneous/plot_metadata_routing.py

```diff
@@ -435,48 +467,61 @@ def __init__(self, transformer, classifier):
         )
         return router
 
     def fit(self, X, y, **fit_params):
-        params = process_routing(self, "fit", **fit_params)
+        routed_params = process_routing(self, "fit", **fit_params)
 
-        self.transformer_ = clone(self.transformer).fit(X, y, **params.transformer.fit)
-        X_transformed = self.transformer_.transform(X, **params.transformer.transform)
+        self.transformer_ = clone(self.transformer).fit(
+            X, y, **routed_params.transformer.fit
+        )
+        X_transformed = self.transformer_.transform(
+            X, **routed_params.transformer.transform
+        )
 
         self.classifier_ = clone(self.classifier).fit(
-            X_transformed, y, **params.classifier.fit
+            X_transformed, y, **routed_params.classifier.fit
         )
         return self
 
     def predict(self, X, **predict_params):
-        params = process_routing(self, "predict", **predict_params)
+        routed_params = process_routing(self, "predict", **predict_params)
 
-        X_transformed = self.transformer_.transform(X, **params.transformer.transform)
-        return self.classifier_.predict(X_transformed, **params.classifier.predict)
+        X_transformed = self.transformer_.transform(
+            X, **routed_params.transformer.transform
+        )
+        return self.classifier_.predict(
+            X_transformed, **routed_params.classifier.predict
+        )
 
 
 # %%
-# Note the usage of :class:`~utils.metadata_routing.MethodMapping` to declare
-# which methods of the child estimator (callee) are used in which methods of
-# the meta estimator (caller). As you can see, we use the transformer's
-# ``transform`` and ``fit`` methods in ``fit``, and its ``transform`` method in
-# ``predict``, and that's what you see implemented in the routing structure of
-# the pipeline class.
+# Note the usage of :class:`~utils.metadata_routing.MethodMapping` to
+# declare which methods of the child estimator (callee) are used in which
+# methods of the meta estimator (caller). As you can see, `SimplePipeline` uses
+# the transformer's ``transform`` and ``fit`` methods in ``fit``, and its
+# ``transform`` method in ``predict``, and that's what you see implemented in
+# the routing structure of the pipeline class.
 #
 # Another difference in the above example with the previous ones is the usage
 # of :func:`~utils.metadata_routing.process_routing`, which processes the input
-# parameters, does the required validation, and returns the `params` which we
-# had created in previous examples. This reduces the boilerplate code a
-# developer needs to write in each meta-estimator's method. Developers are
+# parameters, does the required validation, and returns the `routed_params`
+# which we had created in previous examples. This reduces the boilerplate code
+# a developer needs to write in each meta-estimator's method. Developers are
 # strongly recommended to use this function unless there is a good reason
 # against it.
```

Comment:
Maybe here, we should add if in this example, there was a good reason for not using it before or if this was for demonstration purposes.